misc/config_definitions.json

```diff
@@ -1547,9 +1547,9 @@
         "fping6": {
             "default": "fping6",
             "group": "external",
-            "section": "executable",
+            "section": "binaries",
             "order": 1,
-            "type": "text"
+            "type": "executable"
```

Comment:
This was already fixed in https://github.com/librenms/librenms/pull/12003

Reply:
fork updated :-)